jenkins/docker/docker-copy.jenkinsfile

```diff
@@ -8,7 +8,7 @@ pipeline {
     parameters {
         choice(
             name: 'SOURCE_IMAGE_REGISTRY', 
-            choices: ['opensearchstaging', 'public.ecr.aws/opensearchstaging', 'opensearchproject', 'public.ecr.aws/opensearchproject'],
+            choices: ['opensearchstaging', 'public.ecr.aws/opensearchstaging', 'opensearchproject', 'public.ecr.aws/opensearchproject', "${DATA_PREPPER_STAGING_CONTAINER_REPOSITORY}"],
             description: 'Choose the source image registry'
```

Comment:
@peterzhuamazon lets directly add the registry URL, parameters (drop down) can use external parameters?  would it not add confusion?

Reply:
This is actually to clear the confusion instead of adding.
Directly adding the link of dataprepper ECR with hashes actually did not give much context on what it is.
It would be even worse to maintain later on as no one knows what exactly is that value.

Using env var will not only protect the URL but also give context that this is a data prepper URL.
And does not need to change the jenkins file if the URL get changed.

Thanks.

Reply:
We also used the env var for data prepper staging ECR here. 
https://github.com/opensearch-project/opensearch-build/blob/fe758ac8cbfb264bfbe3f80ef4e4fc530070d6e5/jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile#L74

Reply:
Ya I expected drop down wont parse, the `${}` value and will be just used as `${}` for backend logic, I'm good with this change.